src/llmq/signing_shares.cpp

```diff
@@ -809,7 +809,7 @@ void CSigSharesManager::TryRecoverSig(const CQuorumCPtr& quorum, const uint256&
 
 CDeterministicMNCPtr CSigSharesManager::SelectMemberForRecovery(const CQuorumCPtr& quorum, const uint256 &id, size_t attempt)
 {
-    assert(size_t(attempt) < quorum->members.size());
+    assert(size_t(attempt) <= quorum->members.size());
```

Comment:
Interesting. However if we are going to allow `attempt` to wrap around then `quorum->members.size()` is actually irrelevant here imo, `attempt` should be limited by `recoveryMembers` instead.

From  `src/llmq/params.h`:
```
    // How many members should we try to send all sigShares to before we give up.
    int recoveryMembers;
```

Pls consider 8ff9ba1a41.